milvus/utils/Format.ts

```diff
@@ -53,3 +54,109 @@ export const formatNumberPrecision = (number: number, precision: number) => {
       .join(".")
   );
 };
+
+const LOGICAL_BITS = BigInt(18);
+// const LOGICAL_BITS_MASK = (1 << LOGICAL_BITS) - 1;
+
+const checkTimeParam = (ts: any) => {
+  switch (typeof ts) {
+    case "bigint":
+      return true;
+    case "string":
```

Comment:
Should we merge these two cases like:
```
case "bigint":
case "string":
    return true;
```

Or check string content here?
We could throw an error here if pass a string which contains illegal characters.